lib/internal/assert/assertion_error.js

```diff
@@ -15,6 +15,9 @@ const {
   StringPrototypeSplit,
 } = primordials;
 
+const types = require('internal/util/types');
+const { isNativeError } = types;
```

Comment:
```suggestion
const { isError } = require('internal/util');
```

This also matches errors created differently.